lld/ELF/Arch/TargetImpl.h

```diff
@@ -0,0 +1,94 @@
+//===- TargetImpl.h ---------------------------------------------*- C++ -*-===//
```

Comment:
`//===----------------------------------------------------------------------===//` for new file per https://llvm.org/docs/CodingStandards.html#file-headers